game/static/game/css/board.css

```
@@ -0,0 +1,1135 @@
+    }
+    .header h1 {
+        font-size: 1.5rem;
+        margin: 0;
+    }
+
+    .auth-buttons {
+        position: relative;
+        right: auto;
+        top: auto;
+        gap: 10px;
+        flex-wrap: wrap;
```

Stylelint (17.11.1)

[error] Duplicate property "flex-wrap" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)